jib-core/src/main/java/com/google/cloud/tools/jib/event/JibEventType.java

```diff
@@ -24,6 +26,12 @@
   /** All event types. Handlers for this will always be called first. */
   public static final JibEventType<JibEvent> ALL = new JibEventType<>(JibEvent.class);
 
+  /** Log message event. */
+  public static final JibEventType<LogEvent> LOG = new JibEventType<>(LogEvent.class);
+
+  /** Timer event for timing various part of Jib's execution. */
+  public static final JibEventType<TimerEvent> TIMER = new JibEventType<>(TimerEvent.class);
```

Comment:
Question — since these are related to functionality, maybe they should be LOGGING and TIMING?